### DISCUSSION TITLE
Reproduce behavior of tabs but for spaces

### DISCUSSION THREAD

**Discussion by TheBabu** [lukas-reineke/indent-blankline.nvim#856] [Q&A] [answered]
When my file is using tabs I get an output that looks like this:
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/24832107/814ecb38-fc33-4249-bfe8-8fb02a4dab96)
Which is the expected behavior. However, when I try to use spaces, by replacing each tab with 4 spaces.
I get an output like this:
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/24832107/94e925eb-7fd6-4969-b064-2a4580eb9e5e)
Only when my cursor is in the scope, does the bar actually come up. I was expecting the behavior to be the same as when my file had tabs. I looked in the docs and I noticed the `hooks.builtin.hide_first_space_indent_level` but not sure how to utilize it work if it can actually help.

Any help is appreciated!

**Comment by lukas-reineke** [comment]
The 0 level indentation line shows up for spaces by default as well. If it doesn't, it's caused by something in your config.

**Reply by TheBabu** [reply]
I see, I am using NVChad and they have some options set themselves. How can I force to show the indentation line for spaces to rule that possibility out? Are there any other options that could have this behavior?

**Reply by lukas-reineke** [reply, marked as answer]
There is no option for this, only the hook `hooks.builtin.hide_first_space_indent_level`.

You'll need to look into the NVChad codebase and debug this yourself.

**Reply by TheBabu** [reply]
Got it, I'll ask the NVChad folks. Closing the discussion.
